common/services/prismic/events.js

```diff
@@ -122,8 +122,11 @@ export function parseEventDoc(
               description: interpretation.interpretationType.data.description,
               primaryDescription:
                 interpretation.interpretationType.data.primaryDescription,
+              extraInformation:
+                interpretation.interpretationType.data.primaryDescription,
```

Comment:
is this needed here? also is using primaryDescription?

Reply:
Nice catch!

The `primaryDescription` is as [that is displayed when the interpretation is flagged as primary](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/content/webapp/pages/event.js#L511-L513).

Reply:
removed